src/state/InitState.ts

```diff
@@ -66,6 +68,20 @@ export class InitState implements IState{
         // Check if docker is running and it's the correct version
         const isCorrectDockerComposeVersion = await this.dockerService.isCorrectDockerComposeVersion();
         const isDockerStarted = await this.dockerService.checkDocker();
+        this.dockerService.isPortInUse(this.portsToCheck)
```

Comment:
As it is now after we encounter a port that is busy we will exit the process.
How about if we exit on a later stage and we go through and check all possible ports.  
That way we will output to the user all busy ports if there is more than one.